.circleci/config.yml

```diff
@@ -11,7 +11,7 @@
 
 docker_config: &docker_config
   docker:
-    - image: sas42/ds2-build-env:2018-04-06_15-33-00
+    - image: lanza/ds2-build-env:2018-07-17_17-00-00
```

Comment:
Should be sas42 instead of lanza no? Were you able to upload to his repo?

Reply:
Yea this was submitted before I got permission to push it to estefan's